tests/teal/router/sim_approval_yacc_6.teal

```diff
@@ -0,0 +1,441 @@
+#pragma version 6
```

Comment:
TODO: These `sim_*.teal` fixtures aren't currently utilized because of non-idempotency. If the issue isn't resolved by merge time, these should be removed.